keel-ec2-api/src/main/kotlin/com/netflix/spinnaker/keel/api/ec2/SecurityGroupRule.kt

```diff
@@ -18,9 +18,10 @@ package com.netflix.spinnaker.keel.api.ec2
 import com.netflix.spinnaker.keel.api.schema.Literal
 import com.netflix.spinnaker.keel.api.schema.Optional
 
-sealed class SecurityGroupRule {
+abstract class SecurityGroupRule {
```

Comment:
This change is what caused me to have to manually register subtypes for `SecurityGroupRule` with the object mappers and the `ExtensionRegistry`.